src/components/StatsCard/StatsCard.tsx

```diff
@@ -0,0 +1,41 @@
+const statsCardStyle = 'card bg-base-100 w-full px-3 py-2 h-28 md:h-full';
```

Comment:
Can we use the daisy `Stat` component for this? - https://daisyui.com/components/stat/